app/controllers/courses_controller.rb

```diff
@@ -33,7 +33,11 @@ def index
     @self_accredited_courses = @courses_by_accrediting_provider.delete(@provider.provider_name)
   end
 
-  def show; end
+  def show
+    site_statuses = @course.site_statuses
+    site_statuses = site_statuses.select { |ss| ss.status.in?(%w[running new_status]) }
+    @course_locations = site_statuses.map(&:site).sort_by(&:location_name)
```

Comment:
Is it worth putting this in the decorator as `#running_or_new_locations`?

Reply:
Thought about it but didn't want to extend this PR for a simple bug fix